sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -932,6 +953,17 @@ public static partial class HttpPipelineBuilder
         public static Azure.Core.Pipeline.HttpPipeline Build(Azure.Core.ClientOptions options, params Azure.Core.Pipeline.HttpPipelinePolicy[] perRetryPolicies) { throw null; }
         public static Azure.Core.Pipeline.DisposableHttpPipeline Build(Azure.Core.ClientOptions options, Azure.Core.Pipeline.HttpPipelinePolicy[] perCallPolicies, Azure.Core.Pipeline.HttpPipelinePolicy[] perRetryPolicies, Azure.Core.Pipeline.HttpPipelineTransportOptions transportOptions, Azure.Core.ResponseClassifier? responseClassifier) { throw null; }
         public static Azure.Core.Pipeline.HttpPipeline Build(Azure.Core.ClientOptions options, Azure.Core.Pipeline.HttpPipelinePolicy[] perCallPolicies, Azure.Core.Pipeline.HttpPipelinePolicy[] perRetryPolicies, Azure.Core.ResponseClassifier? responseClassifier) { throw null; }
+        public static Azure.Core.Pipeline.HttpPipeline Build(Azure.Core.Pipeline.HttpPipelineBuildOptions options) { throw null; }
+        public static Azure.Core.Pipeline.DisposableHttpPipeline Build(Azure.Core.Pipeline.HttpPipelineBuildOptions options, Azure.Core.Pipeline.HttpPipelineTransportOptions transportOptions) { throw null; }
+    }
+    public partial class HttpPipelineBuildOptions
+    {
+        public HttpPipelineBuildOptions(Azure.Core.ClientOptions options, Azure.Core.Pipeline.HttpPipelinePolicy[] perCallPolicies, Azure.Core.Pipeline.HttpPipelinePolicy[] perRetryPolicies, Azure.Core.ResponseClassifier? responseClassifier, Azure.RequestFailedDetailsParser? requestFailedDetailsParser = null) { }
+        public Azure.Core.ClientOptions ClientOptions { get { throw null; } set { } }
+        public Azure.Core.Pipeline.HttpPipelinePolicy[] PerCallPolicies { get { throw null; } set { } }
+        public Azure.Core.Pipeline.HttpPipelinePolicy[] PerRetryPolicies { get { throw null; } set { } }
+        public virtual Azure.RequestFailedDetailsParser RequestFailedDetailsParser { get { throw null; } set { } }
```

Comment:
Curious: why is this property `virtual` when others aren't?

Reply:
Good catch - this was not intentional.